server/utils/placementRequests/adminSummary.test.ts

```diff
@@ -128,6 +129,14 @@ describe('adminSummary', () => {
           value: {
             text: apTypeLabels[apType],
           },
+          actions: {
```

Comment:
What's the reasoning behind having the link next the the AP Type?

Reply:
It's a good point - I'm not sure of the reasoning, I agree that it isn't intuitive as the timeline doesn't really relate to the AP type.  
Similar issue here: https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/1584 

@edavey and I met with Sarah last week to look into a more user friendly solution. 
We think it would be good to introduce three tabs on the page for viewing the assessment, application and timeline instead of linking out to each one... But I think I'd struggle to do that this sprint. 
![image](https://github.com/ministryofjustice/hmpps-approved-premises-ui/assets/44123869/703ab34e-150c-4398-b34e-ea7ea3de5d08) 

Another solution could be to add a 'View timeline' link underneath 'View Application' and 'View Assessment' but I assume this design has been approved by stakeholders etc...